src/Utils/Type.php

```diff
@@ -38,6 +38,11 @@ public static function string(): Type
         return new Type(null, 'string');
     }
 
+    public static function int(): Type
```

Comment:
nit: missing docs: The built in int type?

Reply:
done :)